### PR TITLE
Features/jwt token generation

### PR DESCRIPTION
**Pull request by AhmedI999**

### CODE REVIEW

File: src/main/java/com/savicsoft/carpooling/security/JwtUtil.java [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import javax.crypto.SecretKey;
+import java.security.Key;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Component
+public class JwtUtil {
+    private final long EXPIRATION = 360_000_0; // 1 Hour
```
Comment (savicsoft) [inline src/main/java/com/savicsoft/carpooling/security/JwtUtil.java:19]: You should pull this out in the application.properties file so we can change it from there

Comment (AhmedI999) [reply src/main/java/com/savicsoft/carpooling/security/JwtUtil.java:19]: Sorry, I don't get what you mean

Comment (savicsoft) [reply src/main/java/com/savicsoft/carpooling/security/JwtUtil.java:19]: No worries so any configuration we want to move into the application.properties file

Comment (savicsoft) [reply src/main/java/com/savicsoft/carpooling/security/JwtUtil.java:19]: https://www.baeldung.com/properties-with-spring

**Pull request closed without merging**